lib/ts/recipe/accountlinking/types.ts

```diff
@@ -201,5 +201,6 @@ export type AccountInfoWithRecipeId = {
 export type RecipeLevelUser = {
     tenantIds: string[];
     timeJoined: number;
+    timeBanned: number | null;
```

Comment:
i dont think it makes sense to add this on a per recipe level user, instead, just on the top level user object is fine